XpressAI/xircuits
# ✨Enable to do pip installation in different version

# Description

This will allow different version of xircuits when pip installing. Currently have 2 version, **default** and **full**.

## Pull Request Type


## Type of Change


# Tests

1. Build the wheel
   1. `pip install build`
   2. `python -m build`
2. Install the wheel(default) at different folder in new venv
   1. `pip install ..dist/xircuits...whl`
   2. Do `pip list` and check it's the same as `requirement.txt`
3. Install the wheel(full)at different folder in new venv. Don't forget the **`[full]`** at the end.
   1. `pip install ..dist/xircuits...whl[full]`
   2. Do `pip list` and check it's the same as `requirement-full.txt`

## Tested on?


# Notes

Please do add/remove any library in the requirement/requirement-full.txt if required as I'm not sure which library is needed for the default ones and the full version. Thanks you
